app/editor/src/hooks/api-editor/editor/useApiActions.ts

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { defaultEnvelope, extractResponseData, LifecycleToasts } from 'tno-core';
+
+import { IActionModel, useApi } from '..';
+
+/**
+ * Common hook to make requests to the PIMS APi.
+ * @returns CustomAxios object setup for the PIMS API.
+ */
+export const useApiActions = (
```

Comment:
Keep each api hook separate so we don't get a 1,000 line file.  Easier to manage and change without breaking other things.